lib/storage/file.js

```diff
@@ -1061,7 +1061,19 @@ File.prototype.getSignedPolicy = function(options, callback) {
 
   makeAuthorizedRequest_.getCredentials(function(err, credentials) {
     if (err) {
-      callback(err);
+      var signingError = new Error('Signing failed. See `error` property.');
+      signingError.error = err;
+      callback(signingError);
```

Comment:
We could just undo this change and pass the original error? It's only these situations where many API calls are made that the user doesn't realize. When we get a fail along the way, it would likely make no sense to the user.